data/toc.yaml

```diff
@@ -2228,3 +2240,5 @@ FAQ:
         title: Enforcement
       - path: /faq/security/single-sign-on/users-faqs/
         title: Manage users
+  - path: /build/hydrobuild/faq/
```

Comment:
Nice to have: the faqs are listed alphabetically, but i understand having this at the bottom improved visibility